frontend/dispatcher.lua

```diff
@@ -33,8 +33,10 @@ local settingsList = {
     -- Device settings
     show_frontlight_dialog = { category="none", event="ShowFlDialog", title=_("Show frontlight dialog"), device=true, condition=Device:hasFrontlight(),},
     toggle_frontlight = { category="none", event="ToggleFrontlight", title=_("Toggle frontlight"), device=true, condition=Device:hasFrontlight(),},
+    set_frontlight = { category="incrementalnumber", event="SetFlIntensity", min=1, max=Device:getPowerDevice().fl_max, title=_("Set frontlight brightness"), device=true, condition=Device:hasFrontlight(),},
```

Comment:
this should be `category="absolutenumber"` as this cannot be a relative number supplied by a gesture.
min should be 0 to allow setting the FL off.